### ISSUE TITLE
Powermenu does not support accessibility

### ISSUE THREAD

**Issue by lee237592** [skydoves/PowerMenu#81]
**Please complete the following information:**
- Library Version : v2.2.0
- Affected Device(s) : Regardless of manufacturer or os version
 
**Describe the Bug:**

With accessibility turned on, open a menu by long pressing a view that implements powermenu, it cannot be reached with a swipe action.
As the popupwindow does not take accessibility focus, I tried to modify it to forcibly create an accessibility event.
But even in that case, the focus event is duplicated in the view exists outside the menu, and the accessibility action isnt performed normally.

**Expected Behavior:**

If menu was opened, accessibility focus should move to powermenu like other Android common popup window.

**Comment by lee237592** [comment]
We are waiting for your comment.
If you have any plans to support accessibility, Please let us know the schedule.
If not, please let us know how we can fix to support accessibility.

**Comment by sebastienrouif** [comment]
@lee237592 is your problem that the power menu doesn't get "focus" when it opens?

**Comment by skydoves** [comment]
This issue was included in version `2.2.2`. Thanks!

**Comment by cmseo** [comment]
@skydoves 
When accessibility is enabled, swipe gestures should normally move focus only between items inside the power menu, and focus should not escape outside of it. Currently, focus can escape outside the menu, and occasionally, once it has escaped, swipe gestures cannot bring the focus back to the power menu.
